bqn-mode.el

```diff
@@ -6,7 +6,7 @@
 
 ;; Author: Marshall Lochbaum <mwlochbaum@gmail.com>
 ;; Version: 0.0.0
-;; Package-Requires: ((emacs "24") (cl-lib "0.5"))
+;; Package-Requires: ((emacs "24.3"))
```

Comment:
You meant to say "upgrade", not "downgrade", right?

Also, doesn't the code depend on cl-lib?

Otherwise lgtm

Reply:
The version *should* have been `24.4` because of `with-eval-after-load`. So if the dependencies were right in the first place, it would have been a downgrade 😅

The `cl-lib` dependency is implied when having version `24.3` or higher. (I think `cl` was used previously to `cl-lib`, and that `cl-lib` was added as a package downloadable to previous versions).